atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceTableCreationTest.java

```diff
@@ -84,6 +87,9 @@ public void testCreatingMultipleTablesAtOnce() {
             });
         });
 
+        threadPool.shutdown();
+        assertTrue(threadPool.awaitTermination(60, TimeUnit.SECONDS));
```

Comment:
Maybe use `Preconditions.checkState`?

`assertTrue` from JUnit suggests we're testing for correctness rather than verifying that some cleanup action has happened.